test/persona/api/offers/offerImpl/JsonOfferParserSpec.scala

```diff
@@ -0,0 +1,95 @@
+package persona.api.offers.offerImpl
+
+import org.joda.time.DateTime
+import org.junit.runner.RunWith
+import org.specs2.mutable.Specification
+import org.specs2.runner.JUnitRunner
+import persona.api.offer.offerImpl._
+import persona.util.ValidationError
+
+@RunWith(classOf[JUnitRunner])
+class JsonOfferParserSpec extends Specification{
+  "JsonOfferParser" should {
+    "parse json with one criterion" in {
+      val json =
+        """
+          |{
+          |  "creationDay":1437090963326,
+          |  "description":"desc",
+          |  "expirationTime":1437090963326,
+          |  "currentParticipants":2,
+          |  "maxParticipants":10,
+          |  "value":3.50,
+          |  "criteria":[
+          |    {
+          |      "criterionCategory":"criterion1Cat",
+          |      "criterion":"crit"
+          |    }]
+          |}
+        """.stripMargin
+
+            val tryDataSchema = new JsonOfferParser().parse(json)
```

Comment:
Minor: Pick better names for variables.  This returns an offer schema, not a data schema